infer/SigSuggestion.cc

```diff
@@ -145,7 +146,7 @@ void extractSendArgumentKnowledge(core::Context ctx, core::Loc bindLoc, cfg::Sen
         }
         core::TypePtr thisType;
         auto iter = &dispatchInfo;
-        while (iter != nullptr) {
+        while (iter != nullptr && iter->main.method.exists()) {
```

Comment:
This will stop it from iterating past a single non-existent component.

Reply:
We should iterate over all of them

Reply:
Oof. That's what I get for thinking, "Oh, I'll just write a quick fix at midnight."